crates/sui/src/keytool.rs

```diff
@@ -1114,6 +1114,14 @@ impl KeyToolCommand {
                     "https://api.ambrus.studio/callback",
                     &jwt_randomness,
                 )?;
+                let url_14 = get_oidc_url(
+                    OIDCProvider::FanTV,
+                    &eph_pk_bytes,
+                    max_epoch,
+                    "r24bskxyafwwua68et2wmuqeyoa.apps.fantv.world",
+                    "https://fantv-apis.fantiger.com/v1/oauth2/redirect",
```

Comment:
Please see here : https://github.com/sui-foundation/sips/blob/626523fd32ebcdbdb53a73a94634f3f94b153b18/sips/sip-34.md#specification

Reply:
feel free to skip the changes in this file (as whats marked in fastcrypto this is throwing error anyway: https://github.com/MystenLabs/fastcrypto/pull/830/files#diff-81897258a805c29ef8b0319ad0aced2aa3ecef7f654c2489535abb8640ec290dR90)

this was useful for testing in cli, but no need now since you can build frontend for this.